### ISSUE TITLE
How to validate Policy Document best practices.

### ISSUE THREAD

**Issue by bj-1795** [gruntwork-io/terratest#991]
I need to test the policy documents attached to the IAM Role, I found the below link but dont see nay such function iam.go,

https://github.com/gruntwork-io/terratest/pull/130

**Comment by infraredgirl** [comment]
Looks like that PR never got wrapped up and merged, unfortunately. If you'd like to contribute some code to test IAM policy documents, that would be much appreciated!

**Comment by bj-1795** [comment]
Is there any chance to merge this PR? I need this urgently.

**Comment by infraredgirl** [comment]
It looks like the existing PR is quite stale. It hasn't been worked on since 2018, it has merge conflicts and unaddressed review comments. So we can't merge until someone either addresses the issues in the existing PR, or opens an new PR.

**Comment by bj-1795** [comment]
Can you please take this as a feature request.

**Comment by infraredgirl** [comment]
Sure, this feature is definitely something that the codebase could benefit from! Having said that, we don't have any availability to work on this at the moment. So if you'd like to see it done sooner rather than later, a PR would be very appreciated!
